### ISSUE TITLE
Results discrepancy in SPF_NET_DFC deblocking task

### ISSUE THREAD

**Issue by Sailor-t** [leenas233/DFC#2]
Hi,

Thank you for sharing the code for SPF_NET_DFC. I trained the model on the deblocking task using the provided code and followed the instructions in the repository. However, the PSNR-B results I obtained are lower than those reported in the paper:

LIVE1: 27.60 (compared to 28.62 in the paper)
Classic5: 28.41 (compared to 28.61 in the paper)
I would appreciate it if you could clarify whether there are additional details or specific training settings not mentioned in the paper or repository that might explain this discrepancy.

Thank you for your assistance!

**Comment by leenas233** [comment]
> Hi,
> 
> Thank you for sharing the code for SPF_NET_DFC. I trained the model on the deblocking task using the provided code and followed the instructions in the repository. However, the PSNR-B results I obtained are lower than those reported in the paper:
> 
> LIVE1: 27.60 (compared to 28.62 in the paper) Classic5: 28.41 (compared to 28.61 in the paper) I would appreciate it if you could clarify whether there are additional details or specific training settings not mentioned in the paper or repository that might explain this discrepancy.
> 
> Thank you for your assistance!

Thank you for your interest. I use the same training configuration as in the paper and stop early in the fine-tuning phase. May I see your training logs so that I can find the problem?

**Comment by Sailor-t** [comment]
Here it is.  @leenas233 
https://pan.baidu.com/s/1JpwuW4LKXC7EvohSGDXmZA?pwd=v3an

**Comment by Sailor-t** [comment]
> > Here it is. @leenas233 https://pan.baidu.com/s/1JpwuW4LKXC7EvohSGDXmZA?pwd=v3an
> 
> Thanks for reply. How about the performance of your network version of SPF-LUT. This may be caused by different version of SPF-LUT network. The nf of my SPF-LUT network is set to 28

Thank you for your response. Actually, the log I shared are based on the SPF-LUT network version with nf=64.
After further training and evaluation, I obtained PSNR-B values of 27.83 on LIVE1 and 28.69 on Classic5.

**Comment by leenas233** [comment]
> Here it is. @leenas233 https://pan.baidu.com/s/1JpwuW4LKXC7EvohSGDXmZA?pwd=v3an

Thanks for reply. This may be caused by different version of SPF-LUT network. The nf of my SPF-LUT network is set to 28.

By the way, in order to find out the possible problem, may you provide me with more deltails:
1）How about the performance of your network version of SPF-LUT?
2）In your version, can the compressed SPF-LUT achieve comparable performance to the network version?

**Comment by Sailor-t** [comment]
For the network version with nf=64, I achieved PSNR-B values of 27.83 on LIVE1 and 28.69 on Classic5.

For the compresed SPF-LUT, I achieved PSNR-B values of 27.60 on LIVE1 and 28.41 on Classic5. (without finetuning).

**Comment by leenas233** [comment]
> For the network version with nf=64, I achieved PSNR-B values of 27.83 on LIVE1 and 28.69 on Classic5.
> 
> For the compresed SPF-LUT, I achieved PSNR-B values of 27.60 on LIVE1 and 28.41 on Classic5. (without finetuning).

The compressed LUT models in our main results are all finetuned. After fine-tuning, the compressed LUT models can achieve comparable performance to the uncompressed versions.

**Comment by Sailor-t** [comment]
Thank you for your response. 

I realized that I made a modification to the source code: I replaced the color conversion process with OpenCV library functions (e.g., cv2.cvtColor). This might have introduced subtle differences in the data preprocessing step, potentially affecting the results.

To better debug this issue, would it be possible for you to share the trained model you used for the reported results?

**Comment by leenas233** [comment]
> Thank you for your response.
> 
> I realized that I made a modification to the source code: I replaced the color conversion process with OpenCV library functions (e.g., cv2.cvtColor). This might have introduced subtle differences in the data preprocessing step, potentially affecting the results.
> 
> To better debug this issue, would it be possible for you to share the trained model you used for the reported results?

of course, trained models are uploaded in [https://drive.google.com/drive/folders/1xhWqwsZeUs623TX1JnOt7nF1IXETts7u](https://drive.google.com/drive/folders/1xhWqwsZeUs623TX1JnOt7nF1IXETts7u)
